yara/yara2/Makefile.acr

```diff
@@ -40,10 +40,11 @@ mrproper: clean
 install:
 	mkdir -p ${DESTDIR}${LIBDIR}/radare2-extras/${VERSION}/
 	${INSTALL_LIB} ${MOD}.${LIBEXT} $(THEDIR)/${MOD}.${LIBEXT}
-	cd ${THEDIR} ; ln -fs ${VERSION} last
-	rm -rf $(THEDIR)/yara
-	mkdir -p $(THEDIR)/yara
-	cp -f ../rules-yara2/* $(THEDIR)/rules-yara2
+	cp -f ${THEDIR}/${MOD}.${LIBEXT} ${R2PM_PLUGDIR}/${MOD}.${LIBEXT}
+	cd ${THEDIR} ; ln -fs ${VERSION} ../last
+	rm -rf ${R2PM_PLUGDIR}/yara
```

Comment:
i think this stuff can be wrong if there are spaces in the path.. what about quoting and testing that spacy case?